Scripts/script-ParseEmailHeaders.yml

```diff
@@ -0,0 +1,38 @@
+commonfields:
+  id: ParseEmailHeaders
+  version: -1
+name: ParseEmailHeaders
+script: |-
+  from email.parser import HeaderParser
+
+  ec = {"Email.Headers" : {}}
+  md = ""
+
+  # Get the file path of the email file as it lokated in Demisto server
+  filePath = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryid']})[0]['Contents']['path']
+
+  # Read the content of the file
+  f = open(filePath, 'rb')
```

Comment:
You better use `with open((filePath, 'rb') as f`...
or at least close the file afterwards

Reply:
Changed